tools/iothub-explorer/readme.md

```diff
@@ -1,15 +1,19 @@
 #iothub-explorer
-A sample CLI to manage devices using the Azure IoT Hub service SDK.
+A sample CLI tool to manage device clients using the Azure IoT Hub service SDK. The tool enables you to manage the devices in the identity register, monitor device-to-cloud messages, and send cloud-to-device commands.
 
 [![npm version](https://badge.fury.io/js/iothub-explorer.svg)](https://badge.fury.io/js/iothub-explorer)
 
-## Install
+## Installing iothub-explorer
 
-`npm install -g iothub-explorer@latest` to get the latest (pre-release) version.
+To install the latest (pre-release) version of the *iothub-explorer* tool, run the following command in your command-line environment:
 
-## Getting Started
+```
+npm install -g iothub-explorer@latest
+```
 
-Supply your IoT Hub connection string once via `login`:
+## Working with the device identity registry
+
+Supply your IoT Hub connection-string once using the **login** command. This means you do not need to supply the connection-string for subsequent commands for the duration of the session (defaults to one hour):
 
 ```shell
 $ iothub-explorer login "HostName=<my-hub>.azure-devices.net;SharedAccessKeyName=<my-policy>;SharedAccessKey=<my-policy-key>"
```

Comment:
@dominicbetts - Can we add the command line to dump the "help" for iothub-explorer?